src/thor/route_action.cc

```diff
@@ -242,6 +243,7 @@ thor::PathAlgorithm* thor_worker_t::get_path_algorithm(const std::string& routet
     PointLL ll1(origin.ll().lng(), origin.ll().lat());
     PointLL ll2(destination.ll().lng(), destination.ll().lat());
     if (ll1.Distance(ll2) < max_timedep_distance) {
```

Comment:
this line... if the route is more than 500km then it wont do time dependent but instead falls back to bidirastar.